modules/mod_audio_fork/mod_audio_fork.h

```diff
@@ -27,6 +27,7 @@
 #define EVENT_PAUSE            "mod_audio_fork::pause"
 #define EVENT_RESUME           "mod_audio_fork::resume"
```

Comment:
Maintain lint here

Reply:
and Make sure the module is compiling and working fine with other use cases

Reply:
tested to latest. working fine. merging in main branch.